deploy/crds/managed.openshift.io_customdomains_crd.yaml

```diff
@@ -51,16 +51,14 @@ spec:
               description: This field can be used to define the custom domain
               type: string
             scope:
-              default: External
-              description: This field determines whether the CustomDomain ingress is internal or external
+              description: This field determines whether the CustomDomain ingress is internal or external. Defaults to External if empty.
```

Comment:
[✓](https://github.com/openshift/custom-domains-operator/pull/37/files#diff-71f68d48c4d024a88d5d71f8b173da8e4319b5e86ca6901c77e70c4b4240b4a4R225-R227)

Be cool to add a unit test proving the defaulting behavior. But meh.

Reply:
Good idea, will follow up in a new PR.